### PR TITLE
OCPBUGS-28216: Adjust width and spacing so that warning icons are not clipped.

### PR DESCRIPTION
**Pull request by sg00dwin**
https://issues.redhat.com/browse/OCPBUGS-28216
Screenshot 2024-01-25 at 12 14 04 PM

**After** updated

Screenshot 2024-01-26 at 2 37 05 PM
Screenshot 2024-01-26 at 2 37 18 PM

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss [hunk 1]
```diff
@@ -143,6 +143,10 @@
   align-items: center;
   display: flex;
   overflow: hidden;
+
+  .pf-v5-c-icon {
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss:147]: Is it possible to add a custom class so we we're not introducing a PF version dependency?

Comment (sg00dwin) [reply frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss]: Updated the markup to wrap with `<Icon>` component and apply custom style rule.

**Review by rhamilto — APPROVED** [review]
/lgtm

**Pull request merged**